10-setup-htcondor.sh

```diff
@@ -133,3 +217,60 @@ echo
 echo "Will use the following token(s):"
 condor_token_list
 echo
+
+cd $LOCAL_DIR
+
+# gwms files in the correct location
+cp -a /gwms/* .
+mkdir -p .gwms.d/bin 
+for target in cleanup  postjob  prejob  setup  setup_singularity; do
+    mkdir -p .gwms.d/exec/$target
+done
+cp -a /usr/sbin/osgvo-singularity-wrapper condor_job_wrapper.sh
+
+# minimum env to get glideinwms scripts to work
+export glidein_config=$LOCAL_DIR/glidein_config
+export condor_vars_file=$LOCAL_DIR/main/condor_vars.lst
+
+# set some defaults for the glideinwms based scripts
+if [[ -z $OSG_DEFAULT_CONTAINER_DISTRIBUTION ]]; then
+    OSG_DEFAULT_CONTAINER_DISTRIBUTION="70%__opensciencegrid/osgvo-el7:latest 30%__opensciencegrid/osgvo-el8:latest"
+fi
+cat >$glidein_config <<EOF
+ADD_CONFIG_LINE_SOURCE $PWD/add_config_line.source
+CONDOR_VARS_FILE $condor_vars_file
+ERROR_GEN_PATH $PWD/error_gen.sh
+GLIDEIN_SINGULARITY_REQUIRE OPTIONAL
+GLIDEIN_Singularity_Use PREFERRED
+OSG_DEFAULT_CONTAINER_DISTRIBUTION $OSG_DEFAULT_CONTAINER_DISTRIBUTION
+SINGULARITY_IMAGE_RESTRICTIONS None
+EOF
+touch $condor_vars_file
+
+# grab the latests copy of stashcp
+mkdir -p client
+
+if (curl --silent --fail --location --connect-timeout 30 --speed-limit 1024 -o client/stashcp http://stash.osgconnect.net/public/dweitzel/stashcp/current/stashcp) &>/dev/null; then
+    chmod 755 client/stashcp
```

Comment:
What puts "client" in the PATH?

Reply:
That is part of the expected gwms directory structure. We just have to mimic it here to make it all work together.